dev/NavigationView/NavigationView.cpp

```diff
@@ -1342,9 +1467,46 @@ void NavigationView::OnItemClick(const winrt::IInspectable& /*sender*/, const wi
     if (!m_shouldIgnoreNextSelectionChange && DoesSelectedItemContainContent(clickedItem, itemContainer) && !IsSelectionSuppressed(selectedItem))
     {
         RaiseItemInvoked(selectedItem, false /*isSettings*/, itemContainer);
+        ToggleIsExpandedFromItem(itemContainer);
     }
 }
 
+void NavigationView::ToggleIsExpanded(winrt::NavigationViewItem const& item)
```

Comment:
>ToggleIsExpanded [](start = 21, length = 16)

I prefer move ToggleIsExpanded to NavigationViewItem. Look like it's the responsibility of NavigaitonViewItem.